magento/pwa-studio
# [Owls] PB-55: Row Full Width Contents Are Horizontally Positioned Incorrectly In Desktop Viewport

## Description
Contents are not correctly aligning horizontally when inside a Full-Width appearance Row. The contents are aligning to what would be the very edge of a Contained Row. Instead, they should align with the elements inside the contained row. This is fixed by moving the padding from the main row element, to the "inner" row element and is not backwards compatible. A framework to provide a data patch for the backwards incompatible change is introduced in the related PRs: https://github.com/magento/magento2-page-builder/pull/378 and https://github.com/magento/magento2-page-builder-ee/pull/107

## Related Issue





[PB-55](https://jira.corp.magento.com/browse/PB-55) Row Full Width Contents Are Horizontally Positioned Incorrectly In Desktop Viewport

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Create Page Builder content with one contained row containing a button. Duplicate that row and change the appearance from contained to Full Width. 
2. Verify the buttons in each row are aligned
3. Make sure any similar content created previously to this patch is corrected by running the related data patch found in the other PRs.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
